### ISSUE TITLE
Layer2.finance TVL reported incorrectly

### ISSUE THREAD

**Issue by krzkaczor** [krzkaczor/l2beat#62]
Layer2.finance batches deposits in one place and then uses these funds to invest in various DeFi protocols. That's why watching TVL only on the bridge contract ends up greatly underestimating TVL.

CC: @Dominator008, do you have any library that can help us with this?

**Comment by Dominator008** [comment]
Yeah, will need to check TVL for each DeFi strategy Layer2.finance talks to and sum them up.

Will an http endpoint that returns the TVL work?
